pkgs/by-name/ea/easyeffects/package.nix

```diff
@@ -114,6 +114,11 @@ stdenv.mkDerivation rec {
       )
     '';
 
+  patches = [
+    # Remove when lsp-plugins is >= 1.2.17. (https://github.com/wwmm/easyeffects/issues/3394)
+    ./0001-Revert-Merge-pull-request-3340-from-violetmage-viole.patch
```

Comment:
Use fetchpatch

Reply:
This is not an upstream patch and created by me, so I'm not sure where i would fetchpatch from.

Reply:
Github doesn't provide a .patch equivalent for reverting commits; this should be fine

Reply:
As far as I remember, fetchpatch can revert a patch.